atom/browser/ui/cocoa/atom_menu_controller.mm

```diff
@@ -223,8 +233,6 @@ - (void)addItemToMenu:(NSMenu*)menu
       [NSApp setWindowsMenu:submenu];
     else if (role == base::ASCIIToUTF16("help"))
       [NSApp setHelpMenu:submenu];
-    else if (role == base::ASCIIToUTF16("services"))
-      [NSApp setServicesMenu:submenu];
```

Comment:
Can we log a message here instead? Like '_A "submenu" is no longer required for the "Services" menu, it can be safely removed from the menu template._'.